### PR TITLE
add more e2e for WorkloadRebalancer

### PR DESCRIPTION
**Pull request by chaosi-zju**
**What type of PR is this?**

/kind failing-test

**What this PR does / why we need it**:

add more e2e for WorkloadRebalancer

**Which issue(s) this PR fixes**:

Fixes part of #4840

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by chaosi-zju** [comment]
CC @XiShanYongYe-Chang

**Comment by chaosi-zju** [comment]
CC @XiShanYongYe-Chang

File: test/e2e/workloadrebalancer_test.go [hunk 1]
```diff
@@ -190,6 +190,115 @@ var _ = framework.SerialDescribe("workload rebalancer testing", func() {
...
+
+				expectedWorkloads := []appsv1alpha1.ObservedWorkload{
+					{Workload: deployObjRef, Result: appsv1alpha1.RebalanceSuccessful},
+					{Workload: notExistDeployObjRef, Result: appsv1alpha1.RebalanceFailed, Reason: appsv1alpha1.RebalanceObjectNotFound},
+					{Workload: clusterroleObjRef, Result: appsv1alpha1.RebalanceSuccessful},
+				}
+				checkWorkloadRebalancerResult(expectedWorkloads)
+			})
+
+			ginkgo.By("step3: udpate WorkloadRebalancer spec workloads", func() {
```
Comment (chaosi-zju) [inline test/e2e/workloadrebalancer_test.go:288]: typo error `udpate` -> `update`

**Review by XiShanYongYe-Chang — COMMENTED** [review]
It's generally LGTM.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**